app/services/UserService.py

```diff
@@ -31,3 +31,20 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users() -> list[PydanticUserResponse]:
+    """
+    Retrieves all users.
+    """
+    users = await UserInDB.all()
+    return [PydanticUserResponse.from_orm(user) for user in users]  # Use from_orm for each user
+
```

Comment:
My bad.
from_orm is depreciated since Pydantic V2.
Please use model_validate instead.

Reply:
done